src/ui/index.ts

```diff
@@ -101,28 +102,80 @@ export function getNodeId({ attributes }: UiNode) {
   }
 }
 
+export const getNodeInputType = (attr: UiNodeAttributes): string =>
+  attr && "type" in attr ? attr.type : ""
+
+// TODO remove this type once the SDK is updated and has this enum
```

Comment:
Remove todo

Reply:
Since the SDK doesn't have these enums defined we should maybe keep it? Otherwise I can add it to the @ory/client sdk and we could release it and bump it here?

Reply:
It seems the SDK has it here 
https://github.com/ory/sdk/blob/master/spec/client/v0.2.0-alpha.9.json#L3716-L3728

But the latest NPM version is v0.2.0-alpha.4